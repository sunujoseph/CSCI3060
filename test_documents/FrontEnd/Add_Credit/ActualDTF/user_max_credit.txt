06 userMaxCredit   FS 100
