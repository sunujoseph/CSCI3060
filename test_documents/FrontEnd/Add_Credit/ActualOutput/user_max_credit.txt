Enter Username: 
Enter Amount To Add To Account: 
Error: Cannot Have More Than 999,999.00 In Your Account.
Enter Amount To Add To Account: 