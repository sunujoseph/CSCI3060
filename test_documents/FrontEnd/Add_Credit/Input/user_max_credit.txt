login
userMaxCredit
addcredit
1000000
logout

login
admin1
addcredit
userMaxCredit
1000000
logout
