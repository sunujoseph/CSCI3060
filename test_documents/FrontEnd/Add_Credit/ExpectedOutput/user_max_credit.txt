Enter Username: 
Enter Amount To Add To Account: 
Cannot have more than $999,999 in your account.

Enter Username: 
Enter User Name To Add Credit To: 
Enter Amount To Add: 
User cannot have more than $999,999 in their account.